tests/dat/actions/multipleHeaders.js

```diff
@@ -3,6 +3,6 @@ function main() {
         headers: {
             "Set-Cookie": ["a=b", "c=d"]
         },
-        code: 200
+        statusCode: 200
```

Comment:
Is this a breaking change wrt. the users? (The code itself doesn't appear so, just asking for extra confidence)

Reply:
That's what the property should have always been. The 'code' is otherwise ignored.